### ISSUE TITLE
Does the dataset have the occlusion label?

### ISSUE THREAD

**Issue by chun-chun123** [SilvioGiancola/TrackingNet-devkit#14]
Hi,thanks a lot for your sharing.
I want to know if there are occlusion labels in the annotation of the dataset?
Thanks a lot.

**Comment by chun-chun123** [comment]
I found in the paper that the occlusion lable is provided in the testing dataset,how about the training dataset?

**Comment by SilvioGiancola** [comment]
Hi,

The training set does not contain any long-term occlusion, but may be
partially occluded for some sequences.

We do not provide per-frame occlusion information as in real-world tracking
applications, this information is not available, but feel free to develop a
tracker that detect those cases and improve the overall tracking
performances.

We labeled the testing sequences with attributes to provide more detailed
performance metrics. Note that the full occlusion occurs in labeled testing
sequences only for a very few frames.

Silvio

-- 

This message and its contents, including attachments are intended solely 
for the original recipient. If you are not the intended recipient or have 
received this message in error, please notify me immediately and delete 
this message from your computer system. Any unauthorized use or 
distribution is prohibited. Please consider the environment before printing 
this email.

**Comment by chun-chun123** [comment]
Thanks a lot for your answer.
